scripts/zones/Chateau_dOraguille/npcs/Halver.lua

```diff
@@ -110,7 +110,7 @@ function onTrigger(player,npc)
 		-- Mission San D'Oria 2-3 Journey Abroad 
 		elseif(currentMission == JOURNEY_ABROAD and MissionStatus == 11) then
 			player:startEvent(0x01fb);
-		elseif(currentMission == JOURNEY_ABROAD and MissionStatus == 0) then
+		elseif(currentMission == JOURNEY_ABROAD and MissionStatus == 1) then
```

Comment:
I think i found out what the problem is. The door npc of papal chamber screw up everything.
Thanks homie!